rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/WorkflowSubmitter.java

```diff
@@ -66,17 +66,19 @@ public WorkflowSubmitter(Scheduler scheduler) {
      * It also creates a job visualization HTML file.
      *
      * @param workflowFile a workflow file (XML or archive)
-     * @param variables    variables to be replaced on submission
+     * @param variables variables to be replaced on submission
+     * @param genericInfos generic informations to be replaced on submission
      * @return job ID of the job created for the specified workflow and associated variables.
      * @throws JobCreationRestException
      * @throws NotConnectedRestException
      * @throws PermissionRestException
      * @throws SubmissionClosedRestException
      */
-    public JobId submit(File workflowFile, Map<String, String> variables) throws NotConnectedRestException,
-            PermissionRestException, SubmissionClosedRestException, JobCreationRestException {
+    public JobId submit(File workflowFile, Map<String, String> variables, Map<String, String> genericInfos)
+            throws NotConnectedRestException, PermissionRestException, SubmissionClosedRestException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)